MooseFS/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: "3"
+
+services:
+  weba:
+    build:
+      context: ./weba
+    ports:
+      - "8001:5000"
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /var/lib/mfs
+    depends_on:
+      - chunkserver1
+      - chunkserver2
+    # restart: always
+
+  webb:
+    build:
+      context: ./webb
+    ports:
+      - "8002:5000"
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /var/lib/mfs
+    depends_on:
+      - chunkserver1
+      - chunkserver2
```

Comment:
Since web-B and chunkserver-2 are on the same machine, update the dependency to just chunkserver 2. It should not depend on chunkserver 1.